roles/etcd/defaults/main.yml

```diff
@@ -1,11 +1,11 @@
 ---
 etcd_name: "{{ inventory_hostname }}"
-etcd_version: v3.4.7
+etcd_version: v3.5.8
```

Comment:
If we are upgrading the etcd version, we should also make sure that our upgrade ansible role upgrades etcd, together with apiservers

Reply:
I've updated the example update playbook to include the etcd upgrade.

I tested the procedure with 3 masters in HA upgrading using the playbook and all is looking good.